### ISSUE TITLE
errors after update

### ISSUE THREAD

**Issue by Andyauk** [project-unk/ox_inventory_redm#1]
The errors with the suggested changes. 

```
[       script:ox_lib] An update is available for ox_lib (current version: 2.4.0)
[       script:ox_lib] https://github.com/overextended/ox_lib/releases/tag/v2.5.2
[ script:ox_inventory] SCRIPT ERROR: @ox_inventory/setup/dbcleanup.lua:5: ox_inventory was unable to execute a query!
[ script:ox_inventory] Table 'qbcoreframeworkredm_a727eb.ox_inventory' doesn't exist
[ script:ox_inventory] SELECT owner, name FROM ox_inventory WHERE NOT owner = '' []
[ script:ox_inventory] > fn (@ox_inventory/setup/dbcleanup.lua:5)
[ script:ox_inventory] > <unknown> (@oxmysql/dist/build.js:21765)
[ script:ox_inventory] > processTicksAndRejections (node:internal/process/task_queues:96)
[ script:ox_inventory] > async rawQuery (@oxmysql/dist/build.js:21735)
[      script:oxmysql] SCRIPT ERROR in promise (unhandled rejection): Error: ox_inventory was unable to execute a query!
[      script:oxmysql] Table 'qbcoreframeworkredm_a727eb.ox_inventory' doesn't exist
[      script:oxmysql] DELETE FROM ox_inventory WHERE lastupdated < (NOW() - INTERVAL 6 MONTH) OR data = "[]" []
[      script:oxmysql] > <unknown> (@oxmysql/dist/build.js:21766)
[      script:oxmysql] > processTicksAndRejections (node:internal/process/task_queues:96)
[      script:oxmysql] > async rawQuery (@oxmysql/dist/build.js:21735)
[      script:oxmysql] 
```

**Comment by Andyauk** [comment]
i see there is a folder called setup. with the sql. ill give that ago

**Comment by Andyauk** [comment]
seems i forgot the sql. but i do see the 


[       script:ox_lib] An update is available for ox_lib (current version: 2.4.0)
[       script:ox_lib] https://github.com/overextended/ox_lib/releases/tag/v2.5.2
